deploy/import-init.sh

```
@@ -10,14 +10,19 @@ fi
 function import_init() {
   NS=keycloak
   CHART_VERSION=0.0.1-develop
+  KEYCLOAK_SERVICE_NAME=keycloak
 
   helm repo add mosip https://mosip.github.io/mosip-helm
   helm repo update
 
   IAM_HOST=$(kubectl get cm global -o jsonpath={.data.mosip-iam-external-host})
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)